test/routes/acolhidosTest/listaAcolhidos.test.js

```diff
@@ -3,18 +3,40 @@ const listaAcolhidos = require("../../../src/routes/acolhido/listaAcolhidos")
 describe('Quando acesso listaAcolhidos', () => {
     it('Deve mostrar uma lista de nomes', (done) => {
 
-        const acolhido = {
+        const Acolhido = {
             findAll : jest.fn()
         }
 
+        const model = {
+            include: jest.fn()
+        }
+
         const req = {}
         const res = {render : jest.fn()}
-        const acolhidos = [{ nome: 'Leo' }]
-        acolhido.findAll.mockResolvedValue(acolhidos)
+        const acolhidos = [{
+            id: '1', 
+            nome: 'Leo',
+            prescricaos: [{ 
+                validade: '2018-06-08', 
+                updated_at: '2018-06-15',
+                acolhido_id: '1' 
+            }] 
+        }]
+
+        const prescricaos = acolhidos.prescricaos
```

Comment:
[Por que prescricaos em vez de prescricoes?](https://stackoverflow.com/questions/49459596/how-do-plurals-work-in-sequelize) 🤔